statuses.py

```diff
@@ -63,4 +63,6 @@
    "Virtual hugs can be just as good as real hugs.",
    "Be a love pharmacist; dispense hugs like medicine — they are!",
    "A hug in time saves nine 🙂.",
+   "Hey you, i'm talking to you, yes you, i'm sending you a hug",
```

Comment:
I love these 🥰 

A few minor grammar fixes:

The "i"s in "I'm" should be capitalized and a period should be added to the end.

I think it might be improved if instead of a comma after every you there was an exclamation point ("Hey you!"), but that's a style choice so I'll leave it up to you :)

Reply:
Fixed it! I hope you like it.